core/src/main/java/org/apache/iceberg/rest/responses/ConfigResponseParser.java

```diff
@@ -83,6 +88,10 @@ public static ConfigResponse fromJson(JsonNode json) {
               .collect(Collectors.toList()));
     }
 
+    if (json.hasNonNull(IDEMPOTENCY_KEY_LIFETIME)) {
```

Comment:
should we check of "" string too ? i guess post ISO string validation this would not be required

Reply:
the String emptiness check should be part of `getDurationStringOrNull`. So the current version of the code makes sense to me